packages/core/src/content-types/tiptap.ts

```diff
@@ -423,6 +423,18 @@ function convertHtmlToTxt(html: string, wrap = true) {
   });
 }
 
+function preserveParagraphLeadingWhitespaces(html: string) {
+  return html.replace(/<p([^>]*)>([ \t]*)/g, (match, attrs, leadingSpaces) => {
```

Comment:
This is _not_ a good idea. We have 2 options here:

1. Use a proper HTML parser to do this
2. Do this inside `html-to-txt`'s `paragraph` formatter above. (preferred)

Reply:
gone with option 2